frontend/styles/var/fonts.scss

```diff
@@ -1,11 +1,22 @@
 // Font sizes
+$xxxx-small: px-to-rem(6);
 $xxx-small: px-to-rem(10);
 $xx-small: px-to-rem(12);
+$xx-small-plus: px-to-rem(13);
 $x-small: px-to-rem(14);
+$x-small-plus: px-to-rem(15);
 $small: px-to-rem(16);
+$small-plus: px-to-rem(18);
 $medium: px-to-rem(20);
 $large: px-to-rem(24);
 $x-large: px-to-rem(28);
+$xx-large: px-to-rem(32);
+$xxx-large: px-to-rem(36);
+$jumbo: px-to-rem(120);
+$x-jumbo: px-to-rem(150);
+$xx-jumbo: px-to-rem(180);
+
+
```

Comment:
Thanks for the PR and effort behind combing through this!

I like the idea of only using defined variables, but how might we keep these stylesheets focused to the core variables?

Adding this to our internal frontend sync agenda for next Wednesday.

Reply:
Yes. That is where I struggled a bit before trying to assign all the sizes to variables. Eventually, I ran out of combinations and left a few out. Would love to know of a solution to this.